### PR TITLE
Added Italian language support

### PR DESCRIPTION
**Pull request by mcarlo95**
Added Italian translation files

Created po/it.po

### CODE REVIEW

File: po/it.po [hunk 1]
```diff
@@ -0,0 +1,464 @@
+# SOME DESCRIPTIVE TITLE.
+# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
+# This file is distributed under the same license as the PACKAGE package.
+# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
```
Comment (Philip-Scott) [inline po/it.po]: Can you remove these lines above line 4? :) I think those were added automatically by poedit

**Comment by mcarlo95** [comment]
I found myself a couple of wrong capital letters.
I tried to compile but cmake can't find libmarkdown (running on Loki). I will try to figure out why tomorrow.

**Comment by mcarlo95** [comment]
Hi! I updated the translation as you asked.
Tried to compile, it says "No package libmarkdown found" and i can't really find a solution.

**Comment by Philip-Scott** [comment]
You're right, i totally broke Loki support on a past update :/ sorry about that.... But your translation will land once Juno is out :)

**Pull request merged**